manifest.json

```diff
@@ -261,7 +261,7 @@
         "targetDirPrefix": "bootstrap-",
         "bundle": {
           "css": ["css/bootstrap.css"],
-          "fontsPath": "fonts",
+          "fontsPath": "webfonts",
```

Comment:
Does bootstrap even have any fonts now?

Reply:
No bootstrap 4 has no fonts, we can also remove this fonts path Thanks!

Reply:
Not done?